### PR TITLE
UAR-463: Added change links for existing MOs

### PR DESCRIPTION
**Pull request by joshuahigginsonCH**
### JIRA link
https://companieshouse.atlassian.net/browse/UAR-463

### Change description
Added change links for existing MO's

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Review by cdrainCH — APPROVED** [review]

File: views/includes/check-your-answers/update/managing-officers-addresses.html [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
+  {% set sameAddressText = "The correspondence address is the same as the principal or registered office address" %}
+  {% set changeResidentialOrPrincipalAddressHtml = OE_CONFIGS.CHANGE_PRINCIPAL_ADDRESS %}
 {% endif %}
 
 {% set moFormattedResidentialOrPrincipalAddressHtml %}
-  {% set address = moAddress %}
-  {% include "includes/display_address.html" %}
+{% set address = moAddress %}
+{% include "includes/display_address.html" %}
 {% endset %}
```
Comment (corybudd-ch) [inline views/includes/check-your-answers/update/managing-officers-addresses.html:14]: Inside the set should be tabbed in

File: views/includes/check-your-answers/update/managing-officers-addresses.html [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
 
 {% set moFormattedServiceAddressHtml %}
-    {% if moSameAddresses == 1 %}
-       {{ sameAddressText }}
-    {% else %}  
-        {% set address = moServiceAddress %}
-        {% include "includes/display_address.html" %}
-    {% endif %}
+{% if moSameAddresses == 1 %}
```
Comment (corybudd-ch) [inline views/includes/check-your-answers/update/managing-officers-addresses.html:17]: same again here, everything inside set tabbed in once

**Review by corybudd-ch — CHANGES_REQUESTED** [review]
Just a general comment which might need to be fixed in multiple places, instead of leaving comments everywhere...
See the 2nd paramater of CREATE_CHANGE_LINK, the string that is passed through, I'm not sure exactly what its for, it gets set to `visuallyHiddenText` on the HTML element, I presume it is some accesibility thing.

Anyways, everywhere else and on the Register version of check your answers (CYA) page, has something more identifying for this string of text.
For example:

1. You use `Individual managing officer - first name` but on the register version of CYA page the string is set too `Individual managing officer " + moi.first_name + " " + moi.last_name + " - first name`

or

2. You use `Individual managing officer - nationality` where register CYA uses `Individual managing officer " + moi.first_name + " " + moi.last_name + " - nationality`

I think your defs will all have to change to be more specific. I'm guessing its because a user can input multiple MOs and the text will need to be different for each MOs links.

To add: you can follow whats already done for Register's CYA MOs links in these files `views/includes/check-your-answers/managing-officer-corporate.html` and `views/includes/check-your-answers/managing-officer-individual.html`

**Review by corybudd-ch — APPROVED** [review]
nice change 👏

**Review by stephanie-ch — APPROVED** [review]

File: views/includes/check-your-answers/update/managing-officers-addresses.html [hunk 3]
```diff
@@ -1,9 +1,11 @@
 {% if managing_officer_individual %}
-    {% set residentialOrPrincipalAddressText = "Home address" %}
-    {% set sameAddressText = "The correspondence address is the same as the home address" %}
+  {% set residentialOrPrincipalAddressText = "Home address" %}
+  {% set sameAddressText = "The correspondence address is the same as the home address" %}
+  {% set changeResidentialOrPrincipalAddressHtml = OE_CONFIGS.CHANGE_RESIDENTIAL_ADDRESS %}
```
Comment (AndreKramer7654) [inline views/includes/check-your-answers/update/managing-officers-addresses.html:4]: this is not "Html" - just a link so name seems somewhat wrong.

Comment (joshuahigginsonCH) [reply views/includes/check-your-answers/update/managing-officers-addresses.html]: fixed

**Pull request merged**